include/psp2kern/kernel/cpu/atomic.h

```diff
@@ -0,0 +1,125 @@
+/**
+ * \kernelgroup{SceCpu}
+ * \usage{psp2kern/kernel/cpu/atomic.h}
+ */
+
+
+#ifndef _PSP2KERN_KERNEL_CPU_ATOMIC_H_
+#define _PSP2KERN_KERNEL_CPU_ATOMIC_H_
+
+#include <psp2kern/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+int ksceKernelAtomicAdd32AndGet64InHiLoRange(unsigned long long *val, int add_val, int limit);
```

Comment:
just a question, don't we need to change types to `SceInt32` and `SceUInt64`??

Reply:
The SceCpuForKernel::atomic stuff was added straight from the wiki. I can change it to SceInt32/SceInt64 if you wish.

Reply:
yep. can you change them?
i'll merge this after that jobs

Reply:
done